hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -132,6 +135,10 @@ public void setCreationTime(long time) {
     this.creationTime = time;
   }
 
+  public void updateModificationTime() {
+    this.modificationTime = System.currentTimeMillis();
```

Comment:
NIT: Should we use Time.now() which is a wrapper of System.currentTimeMillis consistently across the patch? This will make it easier to maintain the code in the long run. Otherwise, LGTM.

Reply:
Thanks @xiaoyuyao for the review.
Replace it with ```setModificationTime()```, and update the corresponding call.